plugins/com.google.cloud.tools.eclipse.dataflow.ui.test/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTabTest.java

```diff
@@ -170,4 +163,14 @@ public boolean apply(Control control) {
       });
     }
   }
+
+  private static class RunnableExecutor implements Answer<Void> {
```

Comment:
Could you please add javadoc here that this it intended to mock `IRunnableContext#run(fork, cancelable,runnable)`, as implemented by major wizard dialogs (e.g., ILaunchConfigurationDialog).

Reply:
Sure, no problem.

Reply:
The other thing to add is that unlike `IRunnableWithContext` implementations, the runnable will be executed on the active thread.